src/acp/policy.ts

```diff
@@ -149,16 +149,13 @@ export function setPolicy(
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Given a [[Policy]] and a set of [[AccessModes]], return a new Access Policy based on the given
- * Access Policy, but with the given Access Modes allowed on it.
+ * Given a [[Policy]] and a set of [[AccessModes]], return a new Policy based on the given
+ * Policy, but with the given Access Modes allowed on it.
  *
- * @param policy An Access Policy on which to set the modes to allow.
- * @param modes Modes to allow for this Access Policy.
+ * @param policy A Policy on which to set the modes to allow.
```

Comment:
4 references to 'A Policy', all of which I'd change to 'The Policy'.

Reply:
```suggestion
 * @param policy The Policy on which to set the modes to allow.
```